API/api/models.py

```
@@ -1,3 +1,84 @@
+    phone = models.CharField(max_length=20, blank=True, )
+    email = models.EmailField(max_length=255, blank=True, null=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    class Meta:
+        db_table = "users"
+        constraints = [
+            models.UniqueConstraint(
+                fields=["email"],
+                condition=Q(email__isnull=False) & ~Q(email=""),
+                name="uq_user_email_not_blank",
+           ),
+        ]
```

Ruff (0.15.0)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)